proposals/unsafe-evolution.md

```diff
@@ -151,8 +151,18 @@ A delegate type that is marked `unsafe` can only be invoked in an `unsafe` conte
 
 ### `extern`
 
-Because `extern` methods are to native locations that cannot be guaranteed by the runtime, any `extern` method is automatically considered `unsafe`. Even methods that only take `unmanaged` parameters by
-value cannot be safely called by C#, as the calling convention used for the method could be incorrectly specified by the user and must be manually verified by review.
+Because `extern` methods are to native locations that cannot be guaranteed by the runtime, any `extern` method is automatically considered `unsafe`
+if compiled under the updated memory safety rules (i.e., it gets the `RequiresUnsafeAttribute`).
```

Comment:
Is it worth pointing out that this is the only place (so far) where the compiler produces `RequiresUnsafeAttribute` annotation without the user actually writing `unsafe` keyword? It feels like an outlier. Are we happy with that?